build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
         classpath 'com.google.gms:google-services:2.1.0'
     }
 }
+project.ext.fabricKey = "";
```

Comment:
I see. Your build is attempting to build the release candidate as well as the debug version, which is why you're running into this error.

You're right that it needs to be fixed but I'd prefer a less global way of doing it.  

Can you put a fix for this into a separate pull request?  It's tangential to the Orientation Login bugs.